clar2wasm/src/datastore.rs

```diff
@@ -78,6 +85,64 @@ fn height_to_id(height: u32) -> StacksBlockId {
     StacksBlockId(height_to_hashed_bytes(height))
 }
 
+fn height_to_block(height: u32, genesis_time: Option<u64>) -> BlockInfo {
+    let bytes = height_to_hashed_bytes(height);
+    let genesis_time = genesis_time.unwrap_or(0);
+
```

Comment:
It is not clear to me (i.e someone not familiar with internals) why this is done this way? Maybe add a brief comment. Is this a specific method or just for mocking?

Reply:
This whole file is just for mocking and is copied from Clarinet. I'll clean up the commented out code, but otherwise, let's leave it unchanged.

Reply:
In 47c62bb57ed615d6f5dd5ecffdca2731cee83f56 I added to the comment at the top of this file that many of these methods are just mock implementations.